ansible/roles/digidecs/templates/digidecs.service.j2

```diff
@@ -0,0 +1,16 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description=Run the digidecs backend
```

Comment:
Does this imply that the backend starts the frontend as well?

Reply:
No, the backend is static and served by nginx